pnpm-lock.yaml

```diff
@@ -15610,6 +15714,45 @@ packages:
       tslib: 2.5.0
     dev: false
 
+<<<<<<< HEAD
+  /@docusaurus/mdx-loader/2.1.0_ny44vnc5t4rlukh2kzmv2f2kze:
+    resolution: {integrity: sha512-i97hi7hbQjsD3/8OSFhLy7dbKGH8ryjEzOfyhQIn2CFBYOY3ko0vMVEf3IY9nD3Ld7amYzsZ8153RPkcnXA+Lg==}
+    engines: {node: '>=16.14'}
+    peerDependencies:
+      react: ^16.8.4 || ^17.0.0
+      react-dom: ^16.8.4 || ^17.0.0
+    dependencies:
+      '@babel/parser': 7.20.15
+      '@babel/traverse': 7.20.13
+      '@docusaurus/logger': 2.1.0
+      '@docusaurus/utils': 2.1.0_@docusaurus+types@2.1.0
+      '@mdx-js/mdx': 1.6.22
+      escape-html: 1.0.3
+      file-loader: 6.2.0_webpack@5.76.2
+      fs-extra: 10.1.0
+      image-size: 1.0.1
+      mdast-util-to-string: 2.0.0
+      react: 17.0.2
+      react-dom: 17.0.2_react@17.0.2
+      remark-emoji: 2.2.0
+      stringify-object: 3.3.0
+      tslib: 2.5.0
+      unified: 9.2.2
+      unist-util-visit: 2.0.3
+      url-loader: 4.1.1_35ful32yo3wjb53le3l6xb5doy
+      webpack: 5.76.2
+    transitivePeerDependencies:
+      - '@docusaurus/types'
+      - '@swc/core'
+      - esbuild
+      - supports-color
+      - uglify-js
+      - webpack-cli
+    dev: false
+
+>>>>>>> next
+=======
+>>>>>>> adfce3382b036f955bcb995937f1cdbc4d45cb30
```

Comment:
Could yoiu please run pnpm install and commit the pnpm lock file? Seems like there are conflicts here still

Reply:
@scopsy  Resolved pls check